.github/workflows/ci.yml

```diff
@@ -39,9 +39,20 @@ jobs:
       matrix:
         python-version: [3.9, "3.10", 3.11]
         os: [ubuntu-latest, macOS-latest, windows-latest]
+    env:
+      BUCKET_NAME : "bioio-dev-test-resources"
+      AWS_REGION : "us-west-2"
+    permissions:
+      id-token: write   # This is required for requesting the JWT
+      contents: read    # This is required for actions/checkout
 
     steps:
     - uses: actions/checkout@v4
+    - uses: aws-actions/configure-aws-credentials@v3
+      with:
+        role-to-assume: arn:aws:iam::978220035532:role/bioio_github
+        role-session-name: bioio-tifffile-${{ github.sha }}
```

Comment:
is there something about the role-session-name or the hash that restricts the set of files to only relevant ones for bioio-tifffile?

Reply:
Nope! This would have access to all the files in the bucket (i.e. all bioio-devs test files), but when it comes to downloading them for each repository they are packaged separately by quilt.

Reply:
Oh, I see now, in download_test_resources.  Thanks!